great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -963,6 +966,12 @@ def resolve_metric_bundle(
                             selectable.columns().subquery()
                         )
                     ).fetchall()
+                elif (Select and isinstance(selectable, Select)) or (
+                    TextualSelect and isinstance(selectable, TextualSelect)
+                ):
+                    res = self.engine.execute(
+                        sa.select(query["select"]).select_from(selectable.subquery())
+                    ).fetchall()
```

Comment:
Warning was: `SADeprecationWarning: Implicit coercion of SELECT and textual SELECT constructs into FROM clauses is deprecated; please call .subquery() on any Core select or ORM Query object in order to produce a subquery object.`